Jenkinsfile

```diff
@@ -17,26 +17,16 @@ pipeline {
                 sh "./gradlew test -i"
             }
         }
-        stage("Integration Tests") {
-            steps {
-                sh "./gradlew integrationTest"
-            }
-        }
-        stage('Publish') {
+        stage("Enterprise Integration Tests") {
             when {
                 anyOf {
                     branch "master*"
-                    branch "release*"
+                    changeRequest target: "master*"
```

Comment:
Will this only run on PRs opened by us?

Or will it also trigger on PRs created by external people? That would be dangeous.

Reply:
In practice we don't trigger separate builds for PR's on Jenkins anyway, hence I'll remove this. 

A bit unfortunate though that we'll run these tests only after it will be merged to master?

Reply:
Actually I'm thinking about adding this again. 
PR's don't run automatically on Jenkins, but it gives to option to trigger it manually and run these tests for a PR. 

WDYT?

Reply:
It is configured correctly on the Jenkins side to only trust PRs from users with write access to this repository, so you can leave it in.
I think it's better to run the integration tests on PRs than to only run them after a merge to master?